addons/account_payment/models/payment_transaction.py

```diff
@@ -88,7 +88,7 @@ def _compute_reference_prefix(self, provider_code, separator, **values):
             invoice_ids = self._fields['invoice_ids'].convert_to_cache(command_list, self)
             invoices = self.env['account.move'].browse(invoice_ids).exists()
             if len(invoices) == len(invoice_ids):  # All ids are valid
-                prefix = separator.join(invoices.mapped('name'))
+                prefix = separator.join([inv.name or inv.name_placeholder_form for inv in invoices])
```

Comment:
Doesn't seem right, it means that you could have had multiple `/` before?

Reply:
Well... yes ?

Reply:
This "reference prefix" is used to generate a "unique reference for the transaction" in `_compute_reference`.

Pretty sure it's not supposed to change over time. (I'm not an expert in `payment` though)